node/network/approval-distribution/src/tests.rs

```diff
@@ -488,7 +488,7 @@ fn peer_sending_us_the_same_we_just_sent_them_is_ok() {
 
 		// import an assignment related to `hash` locally
 		let validator_index = ValidatorIndex(0);
-		let candidate_index = 0u32;
+		let candidate_index = vec![0u32];
```

Comment:
nit: here and below claimed_indices

Reply:
These tests no longer work, I disabled tests so I can build PR and test on Versi :P I think I'll start fixing tests next.